### PR TITLE
i18n: Update Polish translation

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

File: config/locales/pl.yml [hunk 1]
```diff
@@ -212,7 +214,8 @@ pl:
     new_report:
       body: Użytkownik %{reporter} zgłosił %{target}
       subject: Nowe zgłoszenie na %{instance} (#%{id})
-  application_mailer:
+  application_mailer
```
Comment (Gargron) [inline config/locales/pl.yml]: Missing `:`

**Review by Gargron — CHANGES_REQUESTED** [review]

**Comment by nightpool** [comment]
Needs review from another polish translator/user.

**Comment by ykzts** [comment]
https://github.com/tootsuite/mastodon/commits/master/config/locales/pl.yml

@m4sk1n and @rysiekpl ?

**Comment by m4sk1n** [comment]
Thanks for assigning me to review my translation ;)

File: config/locales/pl.yml [hunk 2]
```diff
@@ -336,8 +339,8 @@ pl:
       body: "%{name} poprosił o możliwość śledzenia Cię"
       subject: 'Prośba o możliwość śledzenia: %{name}'
     mention:
-      body: "%{name} wspomniał Cię w:"
-      subject: "%{name} Cię wspomniał"
+      body: "%{name} wspomniał o Tobie w:"
+      subject: "%{name} wspomniał o Tobie"
```
Comment (ykzts) [inline config/locales/pl.yml]: What is the change in this line?

Comment (m4sk1n) [inline config/locales/pl.yml]: Because it looks more natural for me

**Review by Gargron — APPROVED** [review]

**Pull request merged**